e2e/tools/validator/src/validator/prometheus/__init__.py

```diff
@@ -125,6 +129,20 @@ def mape(actual: npt.ArrayLike, expected: npt.ArrayLike) -> ValueOrError:
         return ValueOrError(value=0, error=str(e))
 
 
+def mean_difference(actual: npt.ArrayLike, expected: npt.ArrayLike) -> ValueOrError:
+    try:
+        return ValueOrError(value=100 * np.abs(np.mean(actual) - np.mean(expected)) / np.mean(actual))
```

Comment:
Is this actually how mean difference is calculated? Searching it up, I get conflicting answers with some stating it's the Mean Absolute Error and others stating that it should follow np.mean(actual) - np.mean(expected) which is what is written here. This looks more like mean difference percentage error?

Reply:
this code is to compare the two means from two time series. This is useful to tell when two time series have different number of data points (e.g. with different sampling timestamps). The mean absolute error is about the error of each pairs (or data points).

Reply:
@rootfs , lets also add to the tests. We already have tests for the mse and mape 
Since this finds `%`, how about calling it `mean_diff_percent` ?

Reply:
yes, that makes sense